Minigames/src/main/resources/messages.properties

```diff
@@ -1,13 +1,13 @@
 lang.info=Minigames default language (English)
-player.join.consMsg=%s joined %s
-player.join.plyMsg=%s has joined %s
+player.join.consMsg=\u0020 %s joined %s
+player.join.plyMsg=\u0020 %s has joined %s
```

Comment:
You shouldnt be using UTF8 Chars in here...formatting is handled by the java code ...this is strictly Text

Reply:
Oh righto, strictly text